client/app/lib/changeteam/index.coffee

```diff
@@ -29,6 +29,6 @@ module.exports = class ChangeTeamView extends kd.ModalView
 
     controller = new ChangeTeamController()
     @addSubView list = controller.getView()
-    controller.on 'ItemsLoaded', ->
+    controller.on 'AllItemsAddedToList', ->
```

Comment:
kdlistviewcontroller doesn't emit `ItemsLoaded` 
@gokmen according to testfile
https://github.com/koding/koding/blob/master/client/app/test/kodinglistcontroller.test.coffee#L329
It should emit `ItemsLoaded`. Am I missing something or this usage is deprecated?